### DISCUSSION TITLE
Samples with Cosmos DB vs. Availability of Cosmos DB

### DISCUSSION THREAD

**Discussion by lechnerc77** [Azure/azure-dev#27] [General]
Hi,
this is partially related to the Azure Developer CLI: currently all samples are targeting Cosmos DB. However, at least from my experience in some regions (Like West Europe - see also [Microsoft Q&A](https://docs.microsoft.com/en-us/answers/questions/631203/unable-to-create-cosmos-db-in-any-region.html?page=3&pageSize=10&sort=oldest)) it is impossible to deploy a Cosmos DB since months. I do of course not know the timeline of the public beta and the GA of the tool, but exclusively focusing on Cosmos DB might cause some negative experience when trying the tool. 

While this is not related to the tool at all, it will impact the first experience (an error message when deploying a template is not a good start). Would it make sense to have some (at least) temporal guidance for the regions to use the templates and/or to add templates that are not connected to Cosmos DB?

Happy to hear your opinion.

Best, Christian

**Comment by DaveBurnisonMS** [comment]
I got several "Sorry, we are experiencing high demand in X region, and cannot fulfill your request at this time." errors as I tried 4 or more regions before giving up.

**Comment by markweitzel** [comment]
I believe I mentioned this on another thread as well... Since we are provisioning resources, we are dependent on the capacity in that region. We are working with the Azure teams to better "predict" the availability in the region. For now, are you able to select another region? To do this, you can use the `azd env set AZURE_LOCATION eastus2` command (or whatever region you'd like)

**Comment by savannahostrowski** [comment]
We are tracking an item around this via https://github.com/Azure/azure-dev/issues/2387
